### PR TITLE
SonicBOOM Release README

### PR DESCRIPTION
**Pull request by jerryz123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-We support the FireSim flow to run BOOM at 90+ MHz on FPGAs on Amazon EC2 F1. Created at the University of California,
-Berkeley in the [Berkeley Architecture Research](https://bar.eecs.berkeley.edu/) group, its focus is to create a high
-performance, synthesizable, and parameterizable core for architecture research.
+The Berkeley Out-of-Order Machine (BOOM) is a synthesizable and parameterizable open source RV64GC RISC-V core written in the [Chisel](https://chisel.eecs.berkeley.edu/) hardware construction language. 
+Created at the University of California,
+Berkeley in the [Berkeley Architecture Research](https://bar.eecs.berkeley.edu/) group, its focus is to create a high performance, synthesizable, and parameterizable core for architecture research.
+The current version of the BOOM microarchitecture ([SonicBOOM, or BOOMv3](https://carrv.github.io/2020/papers/CARRV2020_paper_15_Zhao.pdf)) is performance competitive with commercial high-performance out-of-order cores, such as the Cortex A72, Xuantie 910, or SiFive U8, achieving 6.2 CM/MHz.
+
+![](docs/figures/uarch.png)
```
Comment (abejgonzalez) [inline README.md:11]: This is a very large image. Maybe we can put this side to side with the table and be of a similar size?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,31 @@
...
-We support the FireSim flow to run BOOM at 90+ MHz on FPGAs on Amazon EC2 F1. Created at the University of California,
-Berkeley in the [Berkeley Architecture Research](https://bar.eecs.berkeley.edu/) group, its focus is to create a high
-performance, synthesizable, and parameterizable core for architecture research.
+The Berkeley Out-of-Order Machine (BOOM) is a synthesizable and parameterizable open source RV64GC RISC-V core written in the [Chisel](https://chisel.eecs.berkeley.edu/) hardware construction language. 
+Created at the University of California,
+Berkeley in the [Berkeley Architecture Research](https://bar.eecs.berkeley.edu/) group, its focus is to create a high performance, synthesizable, and parameterizable core for architecture research.
+The current version of the BOOM microarchitecture ([SonicBOOM, or BOOMv3](https://carrv.github.io/2020/papers/CARRV2020_paper_15_Zhao.pdf)) is performance competitive with commercial high-performance out-of-order cores, such as the Cortex A72, Xuantie 910, or SiFive U8, achieving 6.2 CM/MHz.
```
Comment (abejgonzalez) [inline README.md]: I think we should have the other cores linked in this sentence.

File: README.md [hunk 3]
```diff
@@ -41,12 +34,31 @@ file in the top level directory of this repository. This file is mainly used for
...
+```
+
+**Website:** (www.boom-core.org)
+
+**Mailing List** (https://groups.google.com/forum/#!forum/riscv-boom)
+
+## Disclaimer!
 
-BOOM is a work-in-progress and remains in active development.
+BOOM is a work-in-progress and remains in active development.
```
Comment (abejgonzalez) [inline README.md]:
```suggestion
BOOM is a work-in-progress and remains in active development.
```

File: README.md (hunk 2 above)
Comment (abejgonzalez) [reply README.md]: Also, I think it might be worth it to mention that this runs through SPEC06/17. Maybe in the table?

**Comment by abejgonzalez** [comment]
Also we can just use `[ci skip]` on these commits since they are more README related.

File: README.md (hunk 1 above)
Comment (jerryz123) [reply README.md:11]: GH Markdown doesn't natively support image scaling. I don't want to resort to HTML to make this work.

**Review by abejgonzalez — APPROVED** [review]
LGTM. Can you rebase and put `[ci skip]` (without a `-`) on the last commit.

**Pull request merged**